src/main/config/cromwell.conf.ctmpl

```diff
@@ -18,7 +18,7 @@ google {
   authScheme = "service"
 
   serviceAuth {
-    p12File = "{{.Data.service_auth_p12_file}}"
+    pemFile = "{{.Data.service_auth_pem_file}}"
```

Comment:
Oh wait (2): we put it in place for **dev**. I don't know who manages the other environments?